extension/parquet/parquet_extension.cpp

```diff
@@ -1444,7 +1456,7 @@ static void ParquetCopySerialize(Serializer &serializer, const FunctionData &bin
 	                                                                         bind_data.encryption_config, nullptr);
 	serializer.WriteProperty(108, "dictionary_compression_ratio_threshold",
 	                         bind_data.dictionary_compression_ratio_threshold);
-	serializer.WritePropertyWithDefault<optional_idx>(109, "compression_level", bind_data.compression_level);
+	serializer.WritePropertyWithDefault<int64_t>(109, "compression_level", bind_data.compression_level);
```

Comment:
Let's keep this an `optional_idx`, and then explicitly map negative values to very high idx_t values, e.g. ```NumericLimits<int64_t>::Maximum() - [negative_value]```